src/silx/gui/plot/items/marker.py

```diff
@@ -75,6 +78,7 @@ def __init__(self):
 
         self._x = None
         self._y = None
+        self._bgColor: colors.RGBAColorType | numpy.ndarray | None = None
```

Comment:
An array of colors is not supported (at least it is not in the OpenGL backend, and if it's not needed better not to do so:
```suggestion
        self._bgColor: colors.RGBAColorType | None = None
```